README.rst

```diff
@@ -702,83 +683,85 @@ where:
+  it will be normalized to ``major.minor.patch-count``.
+  By default, the version string is the result of ``git describe --tags --long``.
+  If your application is not a git repository, you have to set the ``--version`` option
   explicitly.
 
-* ``--suffix string`` (common for all distribution types) is the result file (or image)
-  name suffix.
+* ``--suffix string`` (common for all distribution types) is the suffix of the
+  resulting  file (or image) name.
 
-* ``--unit-template string`` (used for ``rpm`` and ``deb``) is the path to the template for
-  the ``systemd`` unit file.
+* ``--unit-template string`` (for ``rpm`` and ``deb`` packages only) is
+  the path to the template for the ``systemd`` unit file.
 
-* ``--instantiated-unit-template string`` (used for ``rpm`` and ``deb``) is the path to the
-  template for the ``systemd`` instantiated unit file.
+* ``--instantiated-unit-template string`` (for ``rpm`` and ``deb`` packages only)
+  is the path to the template for the ``systemd`` instantiated unit file.
 
-* ``--stateboard-unit-template string`` (used for ``rpm`` and ``deb``) is the path to the
-  template for the stateboard ``systemd`` unit file.
+* ``--stateboard-unit-template string`` (for ``rpm`` and ``deb`` packages only)
+  is the path to the template for the stateboard ``systemd`` unit file.
 
 * ``--use-docker`` (enforced for ``docker``) forces to build the application in Docker.
 
-* ``--tag strings`` (used for ``docker``) is the tag(s) of the Docker image that results from
-  ``pack docker``.
+* ``--tag strings`` (for ``docker`` only) is the tag(s)
+  of the Docker image that results from ``pack docker``.
 
-* ``--from string`` (used for ``docker``) is the path to the base Dockerfile of the runtime
-  image. Defaults to ``Dockerfile.cartridge`` in the application root.
+* ``--from string`` (for ``docker`` only) is the path to the base Dockerfile of the runtime
+  image. Defaults to ``Dockerfile.cartridge`` in the application directory.
 
-* ``--build-from string`` (common for all distribution types, used for building in Docker) is
-  the path to the base Dockerfile of the build image.
-  Defaults to ``Dockerfile.build.cartridge`` in the application root.
+* ``--build-from string`` (common for all distribution types, used for building in Docker)
+  is the path to the base Dockerfile of the build image.
+  Defaults to ``Dockerfile.build.cartridge`` in the application directory.
 
-* ``--no-cache`` creates build and runtime images with ``--no-cache`` docker flag.
+* ``--no-cache`` creates build and runtime images with the ``--no-cache`` Docker flag.
 
-* ``--cache-from strings`` images to consider as cache sources for both build and
-  runtime images. See ``--cache-from`` flag for ``docker build`` command.
+* ``--cache-from strings`` are the images that work as cache sources for both build and
+  runtime images. See the ``--cache-from`` flag for the ``docker build`` command.
 
```

Comment:
I believe it's a good idea to put looong lists like this in the form of a table. This will make it easier to read. What do you think, @NickVolynkin?